app/src/main/res/menu/menu_reading_options.xml

```diff
@@ -7,6 +7,6 @@
     app:showAsAction="never" />
   <item
     android:id="@+id/action_help"
-    android:title="@string/help"
+    android:title="@string/help_activity_title"
```

Comment:
Actually there is one more sting which is `menu_help` or `help_menu`. I suggest using it here as it will make more sense.

Reply:
Okay will do

Reply:
I had suggested this change only for this file. We still need the `help_activity_title` string.

Reply:
okay, I misunderstood that